core/gather/gatherers/full-page-screenshot.js

```diff
@@ -118,9 +117,16 @@ class FullPageScreenshot extends FRGatherer {
 
     // Now that new resources are (probably) fetched, wait long enough for a layout.
     await context.driver.executionContext.evaluate(waitForDoubleRaf, {args: []});
+  }
 
-    const result = await session.sendCommand('Page.captureScreenshot', {
+  /**
+   * @param {LH.Gatherer.FRTransitionalContext} context
+   * @return {Promise<LH.Artifacts.FullPageScreenshot['screenshot']>}
+   */
+  async _takeScreenshot(context) {
+    const result = await context.driver.defaultSession.sendCommand('Page.captureScreenshot', {
       format: 'webp',
+      fromSurface: false,
```

Comment:
[True by default](https://chromedevtools.github.io/devtools-protocol/tot/Page/#method-captureScreenshot)

If this is true, chrome temporarily resizes the viewable area during the screenshot. I don't think this affects the page content but I turned it off to be safe. Setting this to false also decreased the file size of the screenshot.

Reply:
Seems like a good change to have.

Sorry to be a stickler, but can this be a separate PR? Would be useful in the case where a bisect needs to be done, if not hidden in this unrelated commit message. Also I want to spend some time generating reports with this changed options before approving this PR, so it's a blocker for approval as-is.

Reply:
Also am curious how much it decreased the file size. Is there less margin on the image or something? Would like to see before/after pics